src/panels/KaitoPanel.ts

```diff
@@ -117,12 +119,14 @@ export class KaitoPanelDataProvider implements PanelDataProvider<"kaito"> {
     }
 
     private async handleGenerateWorkspaceRequest(webview: MessageSink<ToWebViewMsgDef>) {
-        // after generate workspace CRD, deploy it.
-        webview.postGetWorkspaceResponse({
-            workspace: {
-                workspace: "workspace CRD yaml",
-            },
-        });
+        // webview.postGetWorkspaceResponse({
+        //     workspace: {
+        //         workspace: "workspace CRD yaml",
+        //     },
+        // });
```

Comment:
💡Observation: Do we need these commented line here? if not lets remove any unwanted commented code please? = )

Reply:
this is taken care in this PR - https://github.com/tejhan/vscode-aks-tools/pull/2/files